experiment/build/gcb_build.py

```diff
@@ -39,84 +42,95 @@
 logger = logs.Logger('builder')  # pylint: disable=invalid-name
 
 
+def _get_buildable_images(fuzzer='', benchmark=''):
+    return docker_images.get_images_to_build([fuzzer], [benchmark])
+
+
 def build_base_images():
     """Build base images on GCB."""
-    _build(get_build_config_file('base-images.yaml'), 'base-images')
+    image_templates = {'base-image': _get_buildable_images()['base-image']}
+    config = generate_cloudbuild.create_cloud_build_spec(image_templates,
+                                                         build_base_images=True)
+    _build(config, 'base-images')
 
 
 def build_coverage(benchmark):
     """Build coverage image for benchmark on GCB."""
     coverage_binaries_dir = exp_path.filestore(
         build_utils.get_coverage_binaries_dir())
-    substitutions = {
-        '_GCS_COVERAGE_BINARIES_DIR': coverage_binaries_dir,
-        '_BENCHMARK': benchmark,
+    substitutions = {'_GCS_COVERAGE_BINARIES_DIR': coverage_binaries_dir}
+    buildable_images = _get_buildable_images(fuzzer='', benchmark=benchmark)
+    image_templates = {
+        name: image
+        for name, image in buildable_images.items()
```

Comment:
When I complained that `image` is not descriptive in a previous review, I meant that it was not descriptive pretty much everytime you used that name not just in the location I commented on. Can you change this usage (and every other where you use image to mean image_spec) please?

Reply:
Sorry, missed it. Changed